TUTORIAL.ipynb

```diff
@@ -757,39 +762,44 @@
     "    # read all its data and create a set of winner names\n",
     "    emmy = next(resources)\n",
     "    emmy_winners = set(\n",
-    "        map(lambda x: x['nominee'], \n",
-    "            filter(lambda x: x['winner'],\n",
-    "                   emmy))\n",
+    "        map(lambda x: x[\"nominee\"], filter(lambda x: x[\"winner\"], emmy))\n",
     "    )\n",
     "\n",
     "    # Oscars are next - \n",
     "    # filter rows based on the emmy winner set\n",
     "    academy = next(resources)\n",
-    "    yield filter(lambda row: (row['Winner'] and \n",
-    "                              row['Name'] in emmy_winners),\n",
-    "                 academy)\n",
+    "    yield filter(\n",
```

Comment:
For consistency, all lines were broken down when necessary and got double quotes instead of single quotes.